README.md

```diff
@@ -18,6 +18,25 @@
 
 This repo provides an implementation of an Ingress Controller for NGINX and NGINX Plus from the people behind NGINX.
 
+---
+
+## Join Our Next Community Call!
+
+We value community input and would love to see you at our next community call. At these calls, we discuss PRs by community members as well as issues, discussions and feature requests.
+
+**When**: Every other Monday at 4 PM IST (Irish Standard Time)
```

Comment:
lets add GMT too, would be easy for people in different timezones

Reply:
Good idea, I've added `GMT` to the table. I've changed the `When:` line to say `Irish time` instead of `IST` also, as Irish time is the source of truth for the meeting time, and `IST` is the name of the timezone in the summer months only